openstates/importers/base.py

```diff
@@ -139,6 +140,12 @@ def __init__(self, jurisdiction_id: str, do_postimport=True) -> None:
         if settings.IMPORT_TRANSFORMERS.get(self._type):
             self.cached_transformers = settings.IMPORT_TRANSFORMERS[self._type]
 
+    def get_all_sessions(self) -> None:
```

Comment:
I think this method should return `self.all_sessions_cache` (rather than ONLY load a value to that property) - that would make it consistent with `get_session()` which returns the requested session